tests/test_frontdoor.py

```diff
@@ -982,3 +984,17 @@ def test_utility_warehouse_to_eon():
     referral = models.Referral.objects.get(session_id=session_id)
     assert referral.supplier.name == "E.ON Next"
     referral.delete()
+
+
+@unittest.mock.patch("help_to_heat.frontdoor.interface.EPCApi", MockEPCApi)
+def test_shell_to_octopus():
```

Comment:
Not going to suggest we attack existing tests here, but at least when adding new ones I think we should try for good naming: can this be more descriptive of what it's actually testing? Something of the form "X should Y [when Z]"?